(4a) Squads for VP/UI/Squads.lua

```diff
@@ -118,14 +123,29 @@ function UpdateMouse(gridX, gridY)
 
 
             if bIsBoxSelecting then
-                --print(string.format("Start plot for box select: (%i,%i)", pSelectBoxStartPlot:GetX(), pSelectBoxStartPlot:GetY()))
-                --print(string.format("End plot for box select: (%i,%i)", currentPlot:GetX(), currentPlot:GetY()))
+                -- Determine box select direction across world wrap using mapW
+                if bBoxSelectingLeftToRight == nil and currentPlot:GetX() ~= pSelectBoxStartPlot:GetX() then
+                    if currentPlot:GetX() > pSelectBoxStartPlot:GetX() then
+                        if currentPlot:GetX() - pSelectBoxStartPlot:GetX() > mapW / 2 then
```

Comment:
`bBoxSelectingLeftToRight = currentPlot:GetX() - pSelectBoxStartPlot:GetX() > mapW / 2` is enough

Reply:
good call,
```
if currentPlot:GetX() > pSelectBoxStartPlot:GetX() then
    bBoxSelectingLeftToRight = currentPlot:GetX() - pSelectBoxStartPlot:GetX() < mapW / 2
else
    bBoxSelectingLeftToRight = pSelectBoxStartPlot:GetX() - currentPlot:GetX() > mapW / 2
end
```
 is much cleaner, I've made this change